lib/utils/matrix_sdk_extensions/event_extension.dart

```diff
@@ -17,10 +17,10 @@ extension LocalizedBody on Event {
         future: downloadAndDecryptAttachment,
       );
 
-  void saveFile(BuildContext context) async {
+  Future<String?> saveFile(BuildContext context) async {
```

Comment:
i see you change the output, but not use it, What does it do ?

Reply:
For me it was more logical to get the path so we can use it in toast / download for preview etc

Reply:
good point